gobblin-service/src/main/java/org/apache/gobblin/service/modules/restli/GobblinServiceFlowExecutionResourceHandlerWithWarmStandby.java

```diff
@@ -91,21 +89,20 @@ public UpdateResponse delete(ComplexResourceKey<org.apache.gobblin.service.FlowS
     String flowName = key.getKey().getFlowName();
     Long flowExecutionId = key.getKey().getFlowExecutionId();
     try {
-      // If an existing resume or kill request is still pending then do not accept this request
-      if (this.dagActionStore.exists(flowGroup, flowName, flowExecutionId.toString())) {
-        DagActionStore.DagActionValue action = this.dagActionStore.getDagAction(flowGroup, flowName, flowExecutionId.toString()).getDagActionValue();
-        this.handleException(flowGroup, flowName, flowExecutionId.toString(),
-            new RuntimeException("There is already a pending " + action + " action for this flow. Please wait to resubmit and wait for"
+      // If an existing kill request is still pending then do not accept this request
+      if (this.dagActionStore.exists(flowGroup, flowName, flowExecutionId.toString(), DagActionStore.DagActionValue.KILL)) {
+        this.handleException(flowGroup, flowName, flowExecutionId.toString(), DagActionStore.DagActionValue.KILL,
+            new RuntimeException("There is already a pending KILL action for this flow. Please wait to resubmit and wait for"
```

Comment:
feels like déjà vu... I believe I just read this code above, but w/ `s/RESUME/KILL/g`.  if so, can't we DRY it up?